dingo/bindings/bindings.cpp

```diff
@@ -120,13 +121,26 @@ double HPolytopeCPP::apply_sampling(int walk_len,
    } else if (method == 5) { // dikin walk {
       uniform_sampling<DikinWalk>(rand_points, HP, rng, walk_len, number_of_points,
                                   starting_point, number_of_points_to_burn);
-   } else if (method == 6) { // dikin walk {
+   } else if (method == 6) { // john walk {
       uniform_sampling<JohnWalk>(rand_points, HP, rng, walk_len, number_of_points,
                                  starting_point, number_of_points_to_burn);
-   } else if (method == 7) { // dikin walk {
+   } else if (method == 7) { // vaidya walk {
       uniform_sampling<VaidyaWalk>(rand_points, HP, rng, walk_len, number_of_points,
                                    starting_point, number_of_points_to_burn);
-   } else {
+   } 
+   else if (method == 8) { // gaussian sampling with gaussian HMC exact walk {
+   NT a = NT(1)/(NT(2)*variance);
```

Comment:
Is the `variance` a user's input?

Reply:
no, variance is set to `1.0` in https://github.com/GeomScale/dingo/pull/68/files#diff-603c575493a3a0457022ea65d6c7e5d78e0e8cea4156c1dfeb2b390c8be3a86aR106 Shouldn't be a function parameter, given by the user?

Reply:
Also @stormaths please fix indentation here and below. The code as it is now I think it should give an error by the Python parser and not even execute...

Reply:
Working on having variance and c (will be renamed to bias_vector) as user parameters. Will change the indentation too, though thankfully the C++ compiler ignores it (the test sampling_no_multiphase passes).

Reply:
Still I do not understand why variance is set to `1.0` above.

Reply:
Oh yeah I am working on making it a user parameter, along with bias_vector